packages/preact-urql/src/hooks/useRequest.ts

```diff
@@ -12,7 +12,7 @@ export const useRequest = (
   return useMemo(() => {
     const request = createRequest(query, variables);
     // We manually ensure reference equality if the key hasn't changed
-    if (prev.current !== undefined && prev.current.key === request.key) {
+    if (prev.current !== undefined && prev.current?.key === request.key) {
```

Comment:
We could relax this check to:

```suggestion
    if (prev.current != null && prev.current.key === request.key) {
```

Reply:
is this potentially a big somewhere in Preact / Preact Compat? I see that we explicitly never have `current: null` set as a possible state for this ref, but since that's the default value I'm wondering whether it's an external bug?

Reply:
This is a bug in latest Preact (solved in master)

Reply:
wait, does that mean we can drop this entirely if Preact solved it upstream?

Reply:
imo, we can include this fix and support the version. It's not needed though but I see no harm in `!= null`